LICENSE.md

```diff
@@ -1,7 +1,7 @@
 
 The MIT License (MIT)
 
-Copyright © 2018 by luk3yx
+Copyright © 2018 by luk3yx and other developers of miniirc
```

Comment:
This should link to [the contributors page](https://github.com/luk3yx/miniirc/graphs/contributors).

Reply:
Fixed